dali/pipeline/operators/fused/crop_mirror_normalize.h

```diff
@@ -54,16 +44,20 @@ kernels::SliceFlipNormalizePermutePadArgs<Dims> GetKernelArgs(
     const std::vector<float> &inv_std_dev) {
   kernels::SliceFlipNormalizePermutePadArgs<Dims> args(slice_shape);
 
-  for (std::size_t d = 0; d < Dims; d++) {
+  for (size_t d = 0; d < Dims; d++) {
     args.anchor[d] = slice_anchor[d];
   }
 
+  int channel_dim_idx = ImageLayoutInfo::ChannelDimIndex(input_layout);
+  assert(channel_dim_idx >= 0);
   if (pad_output) {
-    args.padded_shape[channels_dim(input_layout)] = 4;
+    args.padded_shape[channel_dim_idx] = 4;
```

Comment:
Pad to the smallest power of 2 that can accommodate existing number of channels.
Change the doc string (it says it pads to a multiple of 4, but that's not really necessary (?)).

Reply:
done